ui/templates/base.html

```diff
@@ -62,8 +62,8 @@
         <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
           <ul class="nav navbar-nav">
             {% block nav %}{% endblock %}
-            <!-- <li><a href="#"><i class="fa fa-arrow-circle-o-right"></i> Import Course</a></li> -->
-            <!-- <li><a href="#0" class="btn-taxonomies"><i class="fa fa-cogs"></i> Manage Taxonomies</a></li> -->
+            <li><a href="#"><i class="fa fa-arrow-circle-o-right"></i> Import Course</a></li>
```

Comment:
Ahh. If the slug branch has been merged, then probably `repo.slug` instead of `repo.id`.

@pdpinch